lib/matcher.test.js

```diff
@@ -74,6 +74,7 @@ function propertyMatcherTests(matcher, additionalTests) {
         it("compares with matcher", function() {
             var has = matcher("callback", createMatcher.typeOf("function"));
 
+            // eslint-disable-next-line no-empty-function
```

Comment:
I think you can alternatively write `function () { /* empty */ }`. Just sayin, no need to change here.